src/station_cmd.cpp

```diff
@@ -371,7 +377,7 @@ static Station *GetClosestDeletedStation(TileIndex tile)
 				best_station = st;
 			}
 		}
-	}
+	});
```

Comment:
This threshold update can cause different outcomes based on the k-d tree node structure in some very specific cases. I think the trees are not the same for every client, so it can cause desync in those very specific cases.

Reply:
Good point, will have to order on station ID when distances are equal.